13/umbraco-deploy/getting-started/deploy-settings.md

```diff
@@ -372,6 +380,29 @@ In production this setting shouldn't be changed from it's default value of `fals
 
 If attempting a one-off, large transfer operation, before a site is live, you could set this value to `true`. That would omit the firing and handling of these notifications and remove their performance overhead. Following which you would need to ensure to rebuild the cache and search index manually via the backoffice _Settings_ dashboards.
 
+### Suspensions
+
+Deploy operations suspend scheduled publishing, Examine indexing, document cache and/or signature database update events by default. You can amend this behavior for all supported or specific operations using these settings.
+
+Each setting within this section represents a Deploy operation. For each, the suspensions that are carried out can be amended with one or more of following values:
+
+-  `DiskRead` - `None, ScheduledPublishing, Examine, DocumentCache, All`,
+-  `PartialRestore` - `None, ScheduledPublishing, Examine, DocumentCache, All`,
+-  `Restore` - `None, ScheduledPublishing, Examine, DocumentCache, Signatures|All`,
+-  `Deploy` - `None, ScheduledPublishing, All`,
+-  `Import` - `None, ScheduledPublishing, Examine, DocumentCache, All`,
+-  `Export` - `None, ScheduledPublishing, All`
+
+The default value for all suspension settings is `All`.
+
+So for example if you wanted to remove Examine indexing suspension and resumption during partial restore operations, you could set the following:
+
+```json
+  "Suspensions": {
+    "PartialRestore": "ScheduledPublishing, DocumentCache"
+  }
+```
```

Comment:
And we've added operator overloads on the settings class to make configuration in code very easy to do as well:

```c#
using Umbraco.Cms.Core.Composing;
using Umbraco.Deploy.Core.Configuration.DeployConfiguration;

internal class DeploySuspensionComposer : IComposer
{
    public void Compose(IUmbracoBuilder builder)
        => builder.Services.Configure<DeploySettings>(options =>
        {
            // No suspensions during import
            options.Suspensions.Import = SuspensionOptions.None;

            // No Examine suspensions on all operations (using bitwise negation operator)
            options.Suspensions &= ~SuspensionOptions.Examine;

            // Add scheduled publishing suspension to all operations
            options.Suspensions |= SuspensionOptions.ScheduledPublishing;
        });
}
```

Reply:
Have added this example too.